src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -33,4 +33,10 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
         return Optional.ofNullable(relevantPersons);
     }
 
+    /** 
+     *Returns the feedback message to be shown to the user. 
```

Comment:
Avoid accumulating unrelated changes into a single PR.

Tip: If you are currently on `branch1` and want to create a new `branch2`, you need to switch back to `master` branch before creating `branch2`. If not, changes in `branch1` will appear in the PR created from `branch2` later.